src/modules/configurator/components/ConfigProperty.tsx

```diff
@@ -52,7 +55,7 @@ export function ConfigProperty({
         ) : (
           <div className="w-12 2xl:w-14 h-12 2xl:h-14 bg-light-gray-background-color rounded-[50%] animate-pulse" />
         )}
-        {activePropIndex[propertyName] === index && (
+        {!disabled && activePropIndex[propertyName] === index && (
```

Comment:
You can extract this type of bigger conditions (isSomething)

Reply:
Extracted to isSelected.